source/reference/operator/aggregation/geoNear.txt

```diff
@@ -287,6 +294,136 @@ equal to ``Parks``.
       }
    ])
 
+.. _geoNear_let_example:
+
+$geoNear with the ``let`` option
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this example:
+
+- The ``let`` option is used to set an array value of 
+  ``[-73.99279,40.719296]`` to the variable ``$pt``.
+
+- ``$pt`` is specified as a let option to the ``near`` parameter in the 
+  ``$geoNear`` stage.
+
+.. code-block:: javascript
+   :emphasize-lines: 6,16
+
+   db.places.aggregate(
+   [
+      {
+         "$geoNear":
+         {
+            "near":"$$pt",
+            "distanceField":"distance",
+            "maxDistance":2,
+            "query":{"category":"Parks"},
+            "includeLocs":"dist.location",
+            "spherical":true
+         }
+      }
+   ],
+   {
+      "let":{ "pt": [ -73.99279, 40.719296 ] }
+   }
+   )
+
+The aggregation returns the following:
+
+.. code-block:: javascript
+   :copyable: false
+
+   {
+      _id: ObjectId("61715cf9b0c1d171bb498fd7"),
+      name: 'Sara D. Roosevelt Park',
+      location: { type: 'Point', coordinates: [ -73.9928, 40.7193 ] },
+      category: 'Parks',
+      distance: 1.4957325341976439e-7,
+      dist: { location: { type: 'Point', coordinates: [ -73.9928, 40.7193 ] } }
+   },
+   {
+      _id: ObjectId("61715cf9b0c1d171bb498fd6"),
+      name: 'Central Park',
+      location: { type: 'Point', coordinates: [ -73.97, 40.77 ] },
+      category: 'Parks',
+      distance: 0.0009348548688841822,
+      dist: { location: { type: 'Point', coordinates: [ -73.97, 40.77 ] } }
+   }
+
+.. _geoNear_bounded_let_example:
+
+$geoNear with Bound ``let`` Option
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``let`` option can bind a variable which can be used in a 
+$geoNear query.
+
+In this example, :pipeline:`$lookup` uses:
+
+- ``let`` to define ``$pt``.
+- :pipeline:`$geoNear` in the ``pipeline``.
+- ``$pt`` to define ``near`` in the :pipeline:`$geoNear` pipeline stage.
+
+.. code-block:: javascript
+   :emphasize-lines: 5
+
+      db.places.aggregate( [
+      {
+         $lookup:{
+            from: "places",
+            let: { pt: "$location" },
+            pipeline: [
+               {
+                  $geoNear: {
+                     near: "$$pt",
+                     distanceField: "distance"
+                  }
+               }
+            ],
+            as: "joinedField"
+         }
+      },
+      { 
+         $match: { name: "Sara D. Roosevelt Park" }
+      }
+   ] );
+
+The aggregation returns the following:
+
+.. code-block:: javascript
+   :copyable: false
+
+   {
+    _id: ObjectId("61715cf9b0c1d171bb498fd7"),
```

Comment:
Do these objects need to be indented 1-2 more spaces? I'm wondering why the syntax highlighting isn't working in this code-block.

Reply:
Yes in addition to the syntax not picking up the copy button still renders when copyable : false is used. Corrected.